docs/development-tips.md

```diff
@@ -24,6 +24,10 @@ The structure looks like this:
   file, which is automatically picked up by `@babel/preset-env` inside the
   webpack config, if no `browsers` option is supplied.
 
+#### Adding Images
+- Images should be compressed and optimized before being committed to the repo
```

Comment:
Should we provide a workflow for image optimization? Maybe @Scotchester workflow?